conans/client/generators/cmake_find_package.py

```diff
@@ -96,7 +97,8 @@ class CMakeFindPackageGenerator(Generator):
         set({{ pkg_name }}_{{ comp_name }}_RES_DIRS {{ comp.res_paths }})
         set({{ pkg_name }}_{{ comp_name }}_DEFINITIONS {{ comp.defines }})
         set({{ pkg_name }}_{{ comp_name }}_COMPILE_DEFINITIONS {{ comp.compile_definitions }})
-        set({{ pkg_name }}_{{ comp_name }}_COMPILE_OPTIONS_LIST "{{ comp.cxxflags_list }}" "{{ comp.cflags_list }}")
```

Comment:
removing this variable when using components, as this feature is marked as experimental